<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>术有平面扫描仪v3.0.0.0</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>扫描</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="btn_start">
          <property name="toolTip">
           <string>开始</string>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/img/resource/image/start.png</normaloff>:/new/img/resource/image/start.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_stop">
          <property name="toolTip">
           <string>停止</string>
          </property>
          <property name="text">
           <string>停止</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/img/resource/image/stop.png</normaloff>:/new/img/resource/image/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_save">
          <property name="toolTip">
           <string>保存</string>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/img/resource/image/save.png</normaloff>:/new/img/resource/image/save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>视图</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QToolButton" name="btn_zoomIn">
          <property name="toolTip">
           <string>放大</string>
          </property>
          <property name="text">
           <string>放大</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/img/resource/image/zoom_in.png</normaloff>:/new/img/resource/image/zoom_in.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_zoomOut">
          <property name="toolTip">
           <string>缩小</string>
          </property>
          <property name="text">
           <string>缩小</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/img/resource/image/zoom_out.png</normaloff>:/new/img/resource/image/zoom_out.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_original">
          <property name="toolTip">
           <string>实际大小</string>
          </property>
          <property name="text">
           <string>1：1</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/img/resource/image/zoom_1.png</normaloff>:/new/img/resource/image/zoom_1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_adjust">
          <property name="toolTip">
           <string>适应窗口</string>
          </property>
          <property name="text">
           <string>适应窗口</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/img/resource/image/zoom_fit.png</normaloff>:/new/img/resource/image/zoom_fit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>扫描设置</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="check_correct">
              <property name="text">
               <string>校正</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_montage">
              <property name="text">
               <string>拼接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_doubleExposure">
              <property name="text">
               <string>双倍曝光</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="check_autoSave">
              <property name="text">
               <string>自动保存</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdt_dir">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_dir">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_bmp">
              <property name="text">
               <string>BMP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_jpg">
              <property name="text">
               <string>JPG</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,1,0">
            <item>
             <widget class="QCheckBox" name="check_contour">
              <property name="text">
               <string>轮廓提取</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_black">
              <property name="text">
               <string>黑底</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_white">
              <property name="text">
               <string>白底</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hslider_optimize">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_optimize">
              <property name="text">
               <string>优化0%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>横向</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="spin_plan_h">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="spin_actual_h">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QDoubleSpinBox" name="spin_plan_v">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>纵向</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="spin_scale_h">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>10</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QDoubleSpinBox" name="spin_actual_v">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QDoubleSpinBox" name="spin_scale_v">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>10</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>设计值</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>实际值</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>比例</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="1" column="0">
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="View" name="view" native="true">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Mid">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>97</red>
             <green>97</green>
             <blue>97</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Mid">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>97</red>
             <green>97</green>
             <blue>97</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Mid">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>97</red>
             <green>97</green>
             <blue>97</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_status">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QWidget</extends>
   <header location="global">view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
